android/upload_sourcemap.sh

```diff
@@ -18,17 +18,55 @@ if [ ! "${INSTABUG_APP_TOKEN}" ] || [ -z "${INSTABUG_APP_TOKEN}" ]; then
     echo "Instabug: err: INSTABUG_APP_TOKEN not found. Make sure you've added the SDK initialization line Instabug.start Or added the environment variable INSTABUG_APP_TOKEN"
     exit 0
 else
+    if [ ! "${VERSION_CODE}" ] || [ -z "${VERSION_CODE}" ]; then
```

Comment:
How about prefixing the env. variables our SDK uses? (e.g., `INSTABUG_APP_VERSION_CODE` `INSTABUG_APP_VERSION_NAME`)